framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3543,12 +3589,42 @@ protected void addAliasedAnnotation(
      */
     // aliasName is annotated as @FullyQualifiedName because there is no way to confirm that the
     // name of an external annotation is a canoncal name.
-    protected void addAliasedTypeAnnotation(
+    protected final void addAliasedTypeAnnotation(
```

Comment:
Why did you make many methods `final`? Like many of the framework methods these are protected to allow subclasses to change the behavior.

Reply:
I thought because these are called from the constructor it's better to make them not overwrittable https://stackoverflow.com/a/3404369. I also don't want them to be overwritten because all the list files are parsed in this class. No type system overwrite them too.